### DISCUSSION TITLE
Integration example WebAPI support (openTCS baseline 5.17.1, commadapter.vda5050 version 0.20.0)

### DISCUSSION THREAD

**Discussion by antichecker** [openTCS/opentcs#170] [Q&A] [answered]
Hi, quick question:

does the openTCS integration example (openTCS baseline 5.17.1, commadapter.vda5050 version 0.20.0) support the WebAPI trigger to enable the commadapter.vda5050, akin to the WebAPI adress `/vehicles/{NAME}/commAdapter/enabled` documented at swagger.io?

**Comment by swltr** [comment, marked as answer]
@antichecker Yes, [the OpenAPI documentation belonging to 5.17.1](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/openTCS/opentcs/v5.17.1/openTCS-Documentation/src/docs/service-web-api-v1/openapi.yaml#/Vehicles/put_vehicles__NAME__commAdapter_enabled) describes the behaviour in 5.17.1.

**Reply by antichecker** [reply]
Thanks!
